pageserver/src/http/routes.rs

```diff
@@ -270,6 +292,8 @@ async fn timeline_detail_handler(request: Request<Body>) -> Result<Response<Body
         query_param_present(&request, "include-non-incremental-logical-size");
     check_permission(&request, Some(tenant_id))?;
 
+    let ctx = RequestContext::new(TaskKind::MgmtRequest, DownloadBehavior::Download);
```

Comment:
Same here. Though it may require downloads for size maybe? Still would be good to keep `detail` handler free of side effects

Reply:
Right, logical size calculation needs downloading